src/pages/Portfolio/MyKiva/MyKivaPage.vue

```diff
@@ -194,13 +207,19 @@ const handleBadgeJourneyLevelClicked = payload => {
 	);
 };
 
-const handleBadgeModalClosed = () => {
+const handleBadgeModalClosed = isEarnedSection => {
 	if (state.value === STATE_JOURNEY) {
 		selectedBadgeData.value = undefined;
 		showBadgeModal.value = false;
 		return;
 	}
 
+	if (isEarnedSection) {
```

Comment:
Could line `211` just be: `if (state.value === STATE_JOURNEY || state.value === STATE_EARNED) {`?

Reply:
Never mind, following other comment response.